driver/annotations.md

```diff
@@ -0,0 +1,225 @@
+# Adding UAST Annotations
+
+# Intro
+
+Once you have written the [AST parser](driver/sdk.md) the
+next step is to write the annotation code. This is the Go language code that will
+establish the rules to transform the original AST into the UAST and the
+implementation uses two objects, the ObjectToNoder and the AnnotationRules on two
```

Comment:
I would say that the implementation uses two *types* instead of two objects.

I'd also reference the types by their full name: -
- `github.com/bblfsh/sdk/protocol/native.ObjectToNoder`
- `github.com/bblfsh/sdk/uast/ann.Rule`
and maybe with a link to theirs godocs.